examples/networking/router_route/main-route-to-compute.tf

```diff
@@ -0,0 +1,10 @@
+resource "vkcs_networking_router_route" "route_to_compute_example" {
+  depends_on = [
```

Comment:
we do not need this section here at all
1) we use here router and compute, so tf add dependencies on them automatically
2) the compute resource must depend on router interface

Reply:
done